modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -0,0 +1,146 @@
+
+. Create a Secret for your object storage bucket.
++
+You can do this in the *Administrator* view of the OpenShift web console: go to *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Create a namespace for the *TempoStack* instance that you will create in the next step.
++
+You can do this in the *Administrator* view of the OpenShift web console: go to *Home* -> *Projects* -> *Create Project*.
+
+
+. Create a *TempoStack* instance.
++
+NOTE: You can create multiple *TempoStack* instances in separate namespaces on the same cluster.
++
+You can create a *TempoStack* instance as follows:
+
+.. Go to *Operators* -> *Installed Operators*.
++
+[TIP]
+====
+You can use the *Project:* dropdown list to select the namespace that you created for the *TempoStack* instance in the previous step.
+====
+
+.. Select *TempoStack* -> *Create TempoStack* -> *YAML view*.
+
+.. In the *YAML view*, customize the `TempoStack` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <namespace_from_previous_step>
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: <secret-name> <1>
+      type: <secret-provider> <2>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+<1> The value of `name` in the `metadata` of the Secret.
+<2> The accepted values are `azure` for Azure Blob Storage, `gcs` for Google Cloud Storage, and `s3` for Amazon S3 or MinIO or Red Hat OpenShift Data Foundation.
++
+.TempoStack CR for AWS S3 and MinIO storage
+====
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+  namespace: tempostacknamespace
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: minio-test
+      type: s3
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+The stack deployed in this example is configured to receive Jaeger Thrift over HTTP and OpenTelemetry Protocol (OTLP), which permits visualizing the data with the Jaeger UI.
+====
+
+.. Select *Create*.
+
+link:https://github.com/openshift/openshift-docs/pull/62410/files#r1284356755[Waiting for SME input for the following:]
+
+* Step to verify successful deployment of the TempoStack instance
++
+For example:
+** Verify that pods are created on the namespace?
+** Verify/watch/check somewhere (where?) in the OpenShift web console UI that the stack has stabilized.
+* Is there any Tempo UI console? If so, how does the user open it?
+* Any other verification steps?
```

Comment:
@rubenvp8510 and @IshwarKanse, when you can find the time, could you come up with suggested changes for these lines here?

Reply:
I've left comments in 'distr-tracing-tempo-install-cli.adoc' , we can use the same commands here to verify TempoStack instance and getting the route to access the TempoStack query UI.